Configuration/StandardSequences/python/RawToDigi_Data_cff.py

```diff
@@ -2,10 +2,12 @@
 
 from Configuration.StandardSequences.RawToDigi_cff import *
 
+from EventFilter.EcalRawToDigi.EcalRawDataRecovery_cfi import ecalRawDataRecovery
+
 RawToDigi = cms.Sequence(
                          siPixelDigis
                          +siStripDigis
-                         +ecalDigis
+                         +(ecalRawDataRecovery*ecalDigis)
```

Comment:
modifications in this file will need to be made in a different way in CMSSW_8_0_9 and later